lib/ReactViews/Analytics/ParameterEditor.jsx

```diff
@@ -56,7 +67,15 @@ const ParameterEditor = createReactClass({
         this
       );
       if (defined(editor)) {
-        return editor;
+        return (
+          <div
+            style={{
+              color: this.props.parameter.isValid ? "inherit" : "#ff0000"
```

Comment:
Why are you checking isValid here?

Reply:
It the value isn't valid it makes it red - at some point I'll flesh out the form validation stuff, but I think this is enough for the moment